.github/workflows/test-code-format.yaml

```diff
@@ -0,0 +1,44 @@
+name: test code format
+
+on:
+  pull_request:
+    branches:
+      - main
+      - JETSCAPE-4.0-RC
+      - latessa/code_formatter_action
```

Comment:
Do we want to keep this specific branch in the script? Same with the RC branch. Can't we just enforce this on all branches for pull requests?

Reply:
I think until the PR is ready to merge, it could be useful to run the action on my code_formatter_action branch in case I need to make additional changes before merging.  If it's ready to merge, I can remove the latessa/code_formatter_action branch from the YAML before completing the merge.

Reply:
I think we can merge this to the RC branch.

Reply:
Great.  I will remove the code_formatter_action branch from the YAML and complete the merge.